2017/09/24 23:29:30 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 10.0.0.1, server: my.app, request: "GET / HTTP/1.1", upstream: "fastcgi://211.139.178.49:9000", host: "localhost:8080"
2017/09/24 23:29:49 [error] 7#7: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 10.0.0.1, server: my.app, request: "GET / HTTP/1.1", upstream: "fastcgi://211.139.178.49:9000", host: "localhost:8080"
2017/09/24 23:31:31 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 10.0.0.1, server: my.app, request: "GET / HTTP/1.1", upstream: "fastcgi://10.0.0.3:9000", host: "localhost:8080"
2017/09/24 23:31:33 [error] 7#7: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.0.0.1, server: my.app, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://10.0.0.3:9000", host: "localhost:8080", referrer: "http://localhost:8080/"
2017/09/24 23:31:50 [error] 7#7: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.0.0.1, server: my.app, request: "GET / HTTP/1.1", upstream: "fastcgi://10.0.0.3:9000", host: "localhost:8080"
2017/09/24 23:31:52 [error] 7#7: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 10.0.0.1, server: my.app, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://10.0.0.3:9000", host: "localhost:8080", referrer: "http://localhost:8080/"
